apps/web/components/ui/Notifications/Notifications.vue

```diff
@@ -1,7 +1,14 @@
 <template>
   <div data-testid="notifications" class="sticky float-right w-50 right-2 max-w-[450px] z-[51] top-0 h-0">
     <div v-for="notification of notifications" class="my-2" :key="notification.id">
-      <UiAlert :size="'base'" :variant="notification.type">
+      <UiAlert :size="'base'" :variant="notification.type" class="pl-4 gap-0!">
```

Comment:
```suggestion
      <UiAlert :size="'base'" :variant="notification.type" class="pl-4">
```

I don't think this does anything.